test/utils.test.ts

```diff
@@ -744,6 +744,32 @@ describe('#organizeArguments()', () => {
 			utils(settings).organizeArguments();
 		}).toThrow();
 	});
+
+	test('Complains about unexpected pass-through arguments ', () => {
+		const settings = {
+			...settingsPizzaOrdering,
+			arguments: ['order', 'dine-in', '--', '--passThrough-flag', 'passThroughArgument1', 'passThroughArgument2'],
```

Comment:
```suggestion
			arguments: ['order', 'dine-in', '--', '--pass-through-flag', '--pass-through-option=true', 'pass-through-data'],
```

...can you update these arguments in the other two places, too?